docs/components/cards.md

```diff
@@ -0,0 +1,478 @@
+                        </a>
+                    </p>
+                </article>
+
+                <article class="m-card">
+                    <h2 class="m-card_heading">
+                        <a class="" href="#">
+                            <div class="m-card_icon">{% include icons/car.svg %}</div>
+                            <span>Auto loans</span>
+                        </a>
+                    </h2>
+
+                    <ul class="m-list">
+
+                        <li class="m-list_item">
+                            What effect will shopping for
+                            an auto loan have on my credit?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                        <li class="m-list_item">
+                            I owe more on my current loan than my current
+                            vehicle is worth. What do I need to know
+                            if I buy a new vehicle?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                    </ul>
+
+                    <p class="m-card_footer">
+                        <a href="#">
+                            Learn more about bank accounts
+                        </a>
+                    </p>
+                </article>
+
+                <article class="m-card">
+                    <h2 class="m-card_heading">
+                        <a class="" href="#">
+                            <div class="m-card_icon">{% include icons/credit-card.svg %}</div>
+                            <span>Credit cards</span>
+                        </a>
+                    </h2>
+
+                    <ul class="m-list">
+
+                        <li class="m-list_item">
+                            What is a credit card interest rate? What does APR mean?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                        <li class="m-list_item">
+                            How do I dispute a charge on my credit card bill?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                    </ul>
+
+                    <p class="m-card_footer">
+                        <a href="#">
+                            Learn more about bank accounts
+                        </a>
+                    </p>
+                </article>
+            </div>
+        </div>
+
+
+  - variation_name: Card group - two column
+    variation_description: |-
+        A group of cards broken into two columns.
+        The `o-card-group__column-2` modifier is added.
```

Comment:
For better or worse, all the other DS pages (e.g. [column-dividers](https://cfpb.github.io/design-system/components/column-dividers)) include the dot when mentioning class name, so this should be `.o-card-group__column-2`. And the other class mentions on this page should be updated.

Reply:
Fixed in https://github.com/cfpb/design-system/pull/428/commits/47be3e1c7cd2c09874595f22f0599c6d41505ddb